docs/flows-format.adoc

```diff
@@ -0,0 +1,250 @@
+// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
+:_content-type: REFERENCE
+[id="network-observability-flows-format-specifications_{context}"]
+= Network Flows format specifications
+
+Description::
++
+This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export.
++
+The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki, as _Labels_, unlike _Fields_, must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
```

Comment:
```suggestion
The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki, as _Labels_, unlike _Fields_, which must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
```

Reply:
I'm not sure of that one? In its second part, the sentence is already with subject=Labels and verb=must so I don't think "which" would be correct here? (It could be reformulated as: "This distinction only matters when querying Loki. This is because _Labels_, unlike _Fields_, must be used in etc.". )

Reply:
Okay, I think your reworking, "This distinction only matters when querying Loki. This is because Labels, unlike Fields, must be used in etc." is good, lets go with that.